lua/example_of_dynamic_config_request.lua

```diff
@@ -1,8 +1,4 @@
--- TODO #67 Add examples for other operations. For example:
--- Functions to list the ARP table.
--- Functions to list the ND table.
--- Functions to process the GT policies.
--- ......
+-- TODO #67 Add function to update the GT policy.
```

Comment:
The whole comment above should go as discussed in another comment. By the way, the comment above is really about issue #61.